coala_quickstart/generation/Bears.py

```diff
@@ -57,6 +57,9 @@ def filter_relevant_bears(used_languages,
     candidate_bears = copy.copy(bears_by_lang)
     to_propose_bears = {}
 
+    if args.green_mode:
+        return candidate_bears
```

Comment:
Add tests for this, and other `if` statements in this commit

Reply:
👍